### PR TITLE
Support multiple files random and sequential read in StressWorkerBench

### PR DESCRIPTION
**Pull request by bobbai00**
### What changes are proposed in this pull request?

1. add several additional parameters in `WorkerBenchParameters`
2. support multiple files random read&sequential read benchmark in `StressWorkerBench`
### Why are the changes needed?

In order to comparing the worker's read performance more comprehensively, I switch the single test file to multiple test files, and add two modes of reading these files:

- sequential read: each thread will try to read the entire file of every single test file path. When doing sequential read, `--random-read` shoud NOT be set.
- random read: in the prepare stage, several random offsets and lengths will be generated and recorded. And each thread will try to read each test file according to the offset and length. When doing random read, `--random-read`, `--random-seed`, `--random-max-length` and `--random-min-length` should be set.

### Does this PR introduce any user facing changes?
Nope.

### CODE REVIEW

File: libexec/alluxio-config.sh [hunk 1]
```diff
@@ -38,7 +38,7 @@ ALLUXIO_USER_LOGS_DIR="${ALLUXIO_USER_LOGS_DIR:-${ALLUXIO_LOGS_DIR}/user}"
 if [[ -e "${ALLUXIO_CONF_DIR}/alluxio-env.sh" ]]; then
   . "${ALLUXIO_CONF_DIR}/alluxio-env.sh"
 fi
-
+JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk1.8.0_321.jdk/Contents/Home
```
Comment (beinan) [inline libexec/alluxio-config.sh:41]: I guess you might forget to delete this line.

**Review by beinan — APPROVED** [review]
lgtm, great work!

**Review by beinan — APPROVED** [review]

**Comment by beinan** [comment]
one test failing is not related, this pr did not change any function code

**Comment by beinan** [comment]
alluxio-bot, merge this please.

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**